Hands-on lab/Before the HOL - Cloud-native applications.md

```diff
@@ -25,7 +25,7 @@ The names of manufacturers, products, or URLs are provided for informational pur
 -->
 
 <!-- TOC -->
-<!--
+
```

Comment:
can you keep this TOC comment out? 
LGTM other than this. I will merge after this small fix.